app/views/controllers/observations/show/_collection_numbers.erb

```diff
@@ -1,12 +1,14 @@
+<%# locals: (obs: nil, user: nil) %>
```

Comment:
This is an example of Rails template local variable declaration with supplied defaults.